src/fsharp/absil/illib.fsi

```diff
@@ -206,6 +206,8 @@ module internal List =
 
     val internal allEqual: xs:'T list -> bool when 'T: equality
 
+    val isSingleItem: xs: 'T list -> bool
```

Comment:
I think `isSingleton` is more consistent (`List.isEmpty`, `List.singleton`) but no big deal